packages/cli/src/commands/init.ts

```diff
@@ -281,11 +276,13 @@ async function detectPagesOrAppDir(
     // If so then we return app
     // If not return pages
 
+    const extension = isTypescriptProject ? "tsx" : "js"
```

Comment:
determine extension of the file to lookout for

Reply:
Shouldn't this be "jsx", not "js"?